.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.13.0
+current_version = 2.14.0
```

Comment:
I don’t think we should use `bumpversion` as part of a pull request. Current (best?) practice is to do that as part of the release process.

Reply:
I'll revert the `bumpversion`.

At any rate, we should be releasing a `3.0.0` because of the loss of compatibility with 2.7